server.py

```diff
@@ -32,7 +33,33 @@ def get_shopcarts(id):
 
     return jsonify(message), rc
 
+######################################################################
+# Create a Shopcart
+######################################################################
+@app.route('/shopcarts/<int:id>', methods=['POST'])
+def create_shopcart(id):
```

Comment:
You're still `POST`ing on the id - the point is to `POST` on `/shopcarts` and get the data (like the user id) from the body of the request instead of from the URL. 

Here's the flask docs for this:
http://flask.pocoo.org/docs/0.12/quickstart/#accessing-request-data

Reply:
Thank you for the link! @radhikamattoo 
Done!